packages/code-infra/src/eslint/material-ui/config.mjs

```diff
@@ -453,6 +511,17 @@ export function createCoreConfig(options = {}) {
         // https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-no-target-blank.md#rule-options
         'react/jsx-no-target-blank': ['error', { allowReferrer: true }],
 
+        // Enforce using node: protocol for builtin modules
+        'no-restricted-imports': [
```

Comment:
@brijeshb42 This won't merge well, should we rather use a rule from e.g. `eslint-plugin-n`?